string object_type
float32 detection_confidence
float32 heading
float32 xBoxPixel
float32 yBoxPixel
float32 width
float32 height